src/components/Book.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Book = () => (
+  <div className="bookItems">
+    <div className="bookDetails">
+      <span className="details">Comics</span>
+      <h3 className="title">Black Panther</h3>
+      <span className="name">Stan Lee</span>
+      <div className="actions">
+        <ul className="action">
+          <li>Comments</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </div>
+    </div>
+    <div className="progress">
+      <span>77%</span>
+      <span className="status">Completed</span>
+    </div>
+    <div className="chapter">
+      <span>CURRENT CHAPTER</span>
+      <h3>Chapter 20</h3>
+      <button type="button" className="btn">Update progress</button>
+    </div>
+  </div>
+);
+
+export default Book;
```

Comment:
- [ ] As the requirements say you kindly a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books. Kindly make sure the title and the author are coming from the props.